### ISSUE TITLE
Restrict TF model Names

### ISSUE THREAD

**Issue by scopelf** [merantix/picasso#7]
In: picasso/picasso/ml_frameworks/tensorflow/model.py
line: 29-34

The find pattern, line 32, for the latest .ckpt file finds also *.meta* files(!) , which results in an error in line 47 when restoring the model.

I would suggest excluding *.meta* files in the find pattern. However since "glob" does not support exclusions, I suggest to filter the findings (not really elegant):
(line: 32f)
latest_ckpt_fn = max([i for i in glob.iglob(os.path.join(data_dir, '*.ckpt*')) if not ".meta" in i],
                        key=os.path.getctime)

In addition, line 34 (*latest_ckpt_fn[:latest_ckpt_fn.rfind('.ckpt') + 5]*) should be then deleted. (Model filenames does not always have 5 characters and .ckpt )

**Comment by rhsimplex** [comment]
I can't reproduce the error.  The files in my `data-volume` are:

```
checkpoint
convolutional.ckpt.data-00000-of-00001
convolutional.ckpt.index
convolutional.ckpt.meta
```
Which are the provided examples. I don't have any issues loading the model.  Can you show me how to reproduce this or provide a failing test case? Or the files you're trying to load?

**Comment by scopelf** [comment]
The data volume e.g. for the Multinet looks like this:

model.ckpt-17999
model.ckpt-17999.meta

as latest *latest_ckpt_fn* it finds "model.ckpt-17999.meta" not "model.ckpt-17999" and then on line 34 it cuts the name to "model.ckpt" which is not in the directory.

**Comment by scopelf** [comment]
Hi Ryan,

I answered you on github.

Best,

Filippo

**Comment by rhsimplex** [comment]
Ok, got it, thank you!

**Comment by rhsimplex** [comment]
Marking this resolved by #9
